### PR TITLE
Updates to address issues 62, 63 (chapter 8)

### PR DESCRIPTION
**Pull request by rsahita**

### CODE REVIEW

**Comment by rsahita** [comment]
cc @ozkoyuncu

File: specification/sbi_cove.adoc [hunk 1]
```diff
@@ -393,25 +393,31 @@ This requires the host to make three separate ECALLs to the TSM:
...
+the region. If the region of address space is unpopulated, the page removal
+ECALLs are not needed and the host can insert shared pages into the region on
+the next page-fault.
 
 The calling TVM vCPU is considered blocked until the assignment-change is
-completed. Attempts to run it
-with `sbi_covh_run_tvm_vcpu()` will fail. Any guest page faults taken by other
-TVM vCPUs in the
-invalidated pages continue to be reported to the host.
+completed. Attempts to run it with `sbi_covh_run_tvm_vcpu()` will fail.
```
Comment (ozkoyuncu) [inline specification/sbi_cove.adoc:406]: I don't see any change in this paragraph did I miss something. not sure how it addressed the comment

Comment (rsahita) [reply specification/sbi_cove.adoc:406]: the main change to address the issue was from lines 410-417  - the above was just text wrapping issues I fixed in the same PR.

File: specification/sbi_cove.adoc [hunk 2]
```diff
@@ -393,25 +393,31 @@ This requires the host to make three separate ECALLs to the TSM:
...
+continue to be reported to the host.
+
+Note that the TVM vCPU is blocked until the host completes the conversion to
+shared memory - this sequence may happen in two parts - invalidation of
+references to confidential memory (and address translation cache flushes if any)
+and, the addition of the mapping to shared memory - the host may run the TVM
+vCPU after the first part, and lazily handle the fault for the second part.
+Also the reclamation is of the confidential pages, and the shared memory pages
+provided by the host may be unique from those pages so that host has the option
+to service the request on the TVM synchronously or asynchronously.
```
Comment (rsahita) [inline specification/sbi_cove.adoc:417]: @ozkoyuncu this section has the main text added.

Comment (ozkoyuncu) [reply specification/sbi_cove.adoc:417]: ack. thx

**Pull request merged**